source/agora/common/BitField.d

```diff
@@ -176,6 +178,18 @@ public struct BitField (T = uint)
         }
     }
 
+    /// Overrides default vibe.d serialization
+    public string toString () const pure nothrow @safe
+    {
+        return "[" ~ _storage.map!(num_part => num_part.to!string).join(",") ~ "]";
```

Comment:
`return this._storage.to!string` has the same effect here.

Reply:
I like this, but it only works if I remove the 'nothrow' declaration

Reply:
Ah, strange. I think I'll file a bug for this.

Reply:
In any case, it would be good to provide a sink version like suggested before: 

https://github.com/bpfkorea/agora/pull/1233#discussion_r493465721